<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpsList</class>
 <widget class="QWidget" name="OpsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Layers</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushAppendOp">
       <property name="text">
        <string>Append</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboOp">
       <item>
        <property name="text">
         <string>refineAlphaComp</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
